examples/webgl_postprocessing_pixel.html

```diff
@@ -0,0 +1,186 @@
+
+                var shadowLight3 = shadowLight.clone();
+                shadowLight3.intensity = .1;
+                shadowLight3.position.set( 0, 125, 0 );
+
+                scene.add( hemisphereLight );
+                scene.add( shadowLight );
+                scene.add( shadowLight2 );
+                scene.add( shadowLight3 );
+
+                geomData.push( new THREE.SphereGeometry( 1, 64, 64 ) );
+                geomData.push( new THREE.BoxGeometry( 1, 1, 1 ) );
+                geomData.push( new THREE.ConeGeometry( 1, 1, 32 ) );
+                geomData.push( new THREE.TetrahedronGeometry( 1 ) );
+                geomData.push( new THREE.TorusKnotGeometry( 1, .4 ) );
+
+                var numShapes = 25;
+                group = new THREE.Group();
+
+                for ( var i = 0; i < numShapes; i ++ ) {
+
+                    var geom = geomData[ Math.floor( Math.random() * geomData.length ) ];
+                    var mat = new THREE.MeshPhongMaterial( { color: 0xffffff * ( .5 + .5 * Math.random() ) } );
+                    var mesh = new THREE.Mesh( geom, mat );
+                    var s = 4 + Math.random() * 10;
+                    mesh.scale.set( s, s, s );
+
+                    mesh.position.set( Math.random() - 0.5, Math.random() - 0.5, Math.random() - 0.5 ).normalize();
+                    mesh.position.multiplyScalar( Math.random() * 200 );
+                    mesh.rotation.set( Math.random() * 2, Math.random() * 2, Math.random() * 2 );
+                    group.add( mesh );
+
+                }
+
+                scene.add( group );
+
+                composer = new THREE.EffectComposer( renderer );
+                composer.addPass( new THREE.RenderPass( scene, camera ) );
+
+                pixelPass = new THREE.ShaderPass( THREE.PixelShader );
+                pixelPass.renderToScreen = true;
+                composer.addPass( pixelPass );
+
+                window.addEventListener( 'resize', resize );
+
+                params = {
+                    pixels: 256,
+                    step: 2.5,
+                    postprocessing: true
+                };
+                var gui = new dat.GUI();
+                gui.add( params, 'pixels' ).min( 12.0 ).max( 512 ).step( 5.0 );
+                gui.add( params, 'step' ).min( 1.0 ).max( 5.0 ).step( .25 );
+                gui.add( params, 'postprocessing' );
+
+            }
+
+            function update() {
+
+                controls.update();
+                updateGUI();
+                group.rotation.y += .0015;
+                group.rotation.z += .001;
+
+            }
+
+            function animate() {
+
+                update();
+                if ( params.postprocessing )
+                    composer.render();
+                else
+                    renderer.render( scene, camera );
+                window.requestAnimationFrame( animate );
+
+            }
+
+            init();
+            animate();
```

Comment:
These two function calls are normally placed before the function definitions in`three.js`'s examples.